Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -8183,6 +8184,9 @@ void WebPageProxy::decidePolicyForResponseShared(Ref<WebProcessProxy>&& process,
         m_openedMainFrameName = { };
     }
 
+    auto elapsed = (WallTime::now() - m_pageLoadTiming->navigationStart());
```

Comment:
WebPageLoadTiming's m_navigationStart is never set.  You'll have to do that if you want to use this value.